### PR TITLE
docs: 更新常见问题文档

### PR DESCRIPTION
**Pull request by MistEO**
英语太难了_(:з」∠)_ 大佬帮忙康康 @LYZhelloworld

### CODE REVIEW

File: docs/en/FAQ.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ### The program crashes immediately when I try to run it
 
+- Possible solution 0: Compressed package is incomplete
```
Comment (LYZhelloworld) [inline docs/en/FAQ.md:5]: `The downloaded archive is incomplete.`

File: docs/en/FAQ.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
...
+  - If you are using this software for the first time, please do not download the compressed package with the word `OTA` in the file name, this is for incremental update and cannot be used alone
```
Comment (LYZhelloworld) [inline docs/en/FAQ.md:6]: > If you are using this software for the first time

"If you are the first time using this software"

末尾句号丢了

File: docs/en/FAQ.md [hunk 3]
```diff
@@ -14,10 +17,22 @@
...
 
-- Approach 1: find the installation path of your simulator, where there may be a file named `adb.exe` (or something similar, e.g. `nox_adb.exe`, `HD-adb.exe`, `adb_server.exe`, etc., EXE files with `adb`). Simply choose the file in the connection settings of MAA!
-- Approach 2: please refer to the [Custom Connection](#custom-connection) section.
-- Approach 3: change another simulator, such as [Bluestacks international version](https://www.bluestacks.com/download.html) Nougat 64 bit
-- Approach 4: change the code related. You can find the code [here](../src/MeoAsstGui/Helper/WinAdapter.cs) written in C# which is related to the hook on the simulator process and getting ADB path. Feel free to submit PR to us!
+#### Approach 1
+
+1. Make sure that MAA `Settings` - `Connection Settings` - `adb path` is automatically filled in, if it has been filled in, please ignore this step. If not filled in:
+
+    - Method 1: Find the installation path of your simulator, where there may be a file named `adb.exe` (or something similar, e.g. `nox_adb.exe`, `HD-adb.exe`, `adb_server.exe`, etc., EXE files with `adb`). Simply choose the file in the connection settings of MAA!
```
Comment (LYZhelloworld) [inline docs/en/FAQ.md:24]: "Option 1", "Option 2"

Comment (MistEO) [reply docs/en/FAQ.md:24]: 谢谢大佬_(:з」∠)_

**Comment by MistEO** [comment]
https://github.com/MaaAssistantArknights/MaaAssistantArknights/commit/b5dfdf3fbe5d6a9fba14d63e6a48734d1eadc018

**Pull request merged**